### PR TITLE
Fix Traditional Chinese translation

### PR DESCRIPTION
**Pull request by gregman98**
Fixed some wrong usages and punctuations.

### CODE REVIEW

**Comment by Intelli** [comment]
Looks good at a glance, ideally need someone who can read Chinese to review this though.

As well, can you provide screenshots of `/co help`, `/co help params`, and `/co status` with these modifications?
Some of the punctuation changes made may break the in-game UI.

**Comment by gregman98** [comment]
Is it normal for the /co status text to turn white?

File: lang/zh-tw.yml [hunk 1]
```diff
@@ -1,197 +1,198 @@
...
+DATABASE_LOCKED_3: "若要關閉資料庫鎖定功能，請設定 \"database-lock: false\"。"
+DATABASE_LOCKED_4: "關閉資料庫鎖定功能，可能導致資料損壞。"
+DATABASE_UNREACHABLE: "資料庫無法存取，正在拋棄資料並關閉。"
+DEVELOPMENT_BRANCH: "檢測到正在使用開發分支的版本，已跳過使用補丁腳本。"
+DIRT_BLOCK: "已在你的下方放置了泥土。"
+DISABLE_SUCCESS: "執行成功 ！ 關閉 {0} ！"
+ENABLE_FAILED: "無法啟用 {0}。"
+ENABLE_SUCCESS: "已經成功啟用 {0} ！"
+ENJOY_COREPROTECT: "喜歡 {0} 嗎 ？ 加入我們的 Discord ！"
+FINISHING_CONVERSION: "正在完成資料轉換，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:29]:
```suggestion
FINISHING_CONVERSION: "正在完成資料轉換，請稍候……"
```
`稍後` means `a moment later`. e.g. `稍後再試`
`稍候` means `wait a moment`. e.g. `請稍候`

File: lang/zh-tw.yml [hunk 2]
```diff
@@ -1,197 +1,198 @@
...
+FINISHING_LOGGING: "正在完成資料記錄，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:30]:
```suggestion
FINISHING_LOGGING: "正在完成資料記錄，請稍候……"
```

File: lang/zh-tw.yml [hunk 3]
```diff
@@ -1,197 +1,198 @@
...
+INVALID_SELECTION: "未找到選擇 {0} 的內容。"
+INVALID_USERNAME: "\"{0}\" 是無效的玩家名稱。"
+INVALID_WORLD: "請指定有效的世界。"
+LATEST_VERSION: "最新版本 ： {0}"
+LINK_DISCORD: "Discord ： {0}"
+LINK_DOWNLOAD: "下載 ： {0}"
+LINK_PATREON: "Patreon ： {0}"
+LINK_WIKI_BLOCK: "方塊名稱 ： {0}"
+LINK_WIKI_ENTITY: "實體名稱 ： {0}"
+LOGGING_ITEMS: "還剩 {0} 項要進行記錄，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:103]:
```suggestion
LOGGING_ITEMS: "還剩 {0} 項要進行記錄，請稍候……"
```

File: lang/zh-tw.yml [hunk 4]
```diff
@@ -1,197 +1,198 @@
...
-LOOKUP_SEARCHING: "尋找結果中，請稍後..."
-LOOKUP_STORAGE: "{0} {存|取} {1} {2}."
+LOOKUP_CONTAINER: "{0} {新增|刪除} {1} {2}。"
+LOOKUP_HEADER: "{0} 搜尋結果。"
+LOOKUP_INTERACTION: "{0} {點擊|殺死} {1}。"
+LOOKUP_ITEM: "{0} {撿起|拋棄} {1} {2}。"
+LOOKUP_LOGIN: "{0} 已登{入|出}。"
+LOOKUP_PAGE: "第 {0} 頁"
+LOOKUP_ROWS_FOUND: "已找到 {0} {行|行}。"
+LOOKUP_SEARCHING: "正在搜尋，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:113]:
```suggestion
LOOKUP_SEARCHING: "正在搜尋，請稍候……"
```

File: lang/zh-tw.yml [hunk 5]
```diff
@@ -1,197 +1,198 @@
...
+NO_RESULTS_PAGE: "未找到該頁面相關的{結果|資料}。"
+NO_ROLLBACK: "未找到{待處理|上一個}回滾/恢復。"
+PATCH_INTERRUPTED: "更新已中斷 ！ 將會在下次重新啟動時再次嘗試。"
 PATCH_OUTDATED_1: "無法更新早於 {0} 的資料庫。"
-PATCH_OUTDATED_2: "請使用受到支援的CoreProtect版本。"
-PATCH_PROCESSING: "處理新資訊中，請稍後..."
-PATCH_SKIP_UPDATE: "已在 {0} 上跳過 {列表|索引} {更新|新建|刪除}。"
-PATCH_STARTED: "執行 {0} 更新中，請稍後..."
+PATCH_OUTDATED_2: "請使用受到支援的 CoreProtect 版本。"
+PATCH_PROCESSING: "正在處理新資訊，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:135]:
```suggestion
PATCH_PROCESSING: "正在處理新資訊，請稍候……"
```

File: lang/zh-tw.yml [hunk 6]
```diff
@@ -1,197 +1,198 @@
...
+PATCH_SKIP_UPDATE: "已在 {0} 上跳過{列表|索引}的{更新|建立|刪除}。"
+PATCH_STARTED: "正在執行 {0} 更新，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:137]:
```suggestion
PATCH_STARTED: "正在執行 {0} 更新，請稍候……"
```

File: lang/zh-tw.yml [hunk 7]
```diff
@@ -1,197 +1,198 @@
...
 PATCH_SUCCESS: "已成功升級到 {0}。"
-PATCH_UPGRADING: "正在進行資料庫升級中，請稍後..."
-PLEASE_SELECT: "請選擇 \"{0}\" 或是 \"{1}\"。"
+PATCH_UPGRADING: "正在進行資料庫升級，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:139]:
```suggestion
PATCH_UPGRADING: "正在進行資料庫升級，請稍候……"
```

File: lang/zh-tw.yml [hunk 8]
```diff
@@ -1,197 +1,198 @@
...
-ROLLBACK_LENGTH: "所用時間: {0} {秒|秒}。"
-ROLLBACK_MODIFIED: "{更動|更動} {0}。"
-ROLLBACK_RADIUS: "範圍: {0} {方塊|方塊}。"
-ROLLBACK_SELECTION: "範圍設為 \"{0}\"。"
-ROLLBACK_STARTED: "{回朔|恢復|預覽} 開始於 \"{0}\"。"
-ROLLBACK_TIME: "時間軸 {0}。"
-ROLLBACK_WORLD_ACTION: "限制於 \"{0}\"."
+PURGE_NOTICE_1: "請注意，此操作可能需要一些時間。"
+PURGE_NOTICE_2: "在完成此程序前請不要重新啟動伺服器。"
+PURGE_OPTIMIZING: "正在最佳化資料庫，請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:152]:
```suggestion
PURGE_OPTIMIZING: "正在最佳化資料庫，請稍候……"
```

File: lang/zh-tw.yml [hunk 9]
```diff
@@ -1,197 +1,198 @@
...
+PURGE_PROCESSING: "正在處理 {0} 資料……"
+PURGE_REPAIRING: "正在嘗試修復此問題，這可能需要一些時間。"
+PURGE_ROWS: "已刪除 {0} {行|行}資料。"
+PURGE_STARTED: "資料刪除開始於 \"{0}\"。"
+PURGE_SUCCESS: "成功清除資料。"
+RELOAD_STARTED: "正在重新載入配置檔案 - 請稍後……"
```
Comment (jyhsu2000) [inline lang/zh-tw.yml:158]:
```suggestion
RELOAD_STARTED: "正在重新載入配置檔案 - 請稍候……"
```

**Review by jyhsu2000 — CHANGES_REQUESTED** [review]
Some typo found.

`稍後` means `a moment later`. e.g. `稍後再試`
`稍候` means `wait a moment`. e.g. `請稍候`

**Comment by gregman98** [comment]
Thanks for checking. I didn't notice the difference between "稍後" and "稍候".

**Comment by Intelli** [comment]
> Is it normal for the /co status text to turn white?

The formatting issues in `/co status` seem to be due to replacing `:` with `：`. As they look very similar in-game, the easiest solution here is probably just to leave them as `:`.

**Pull request merged**